website/docs/usage/training.md

```diff
@@ -729,13 +729,14 @@ def custom_logger(log_path):
                 file_.write(f"loss_{pipe}\\t")
             file_.write("\\n")
 
```

Comment:
We also need to add the `stdout` and `stderr` arguments to the `setup_logger` function in the example above

Reply:
They're optional, so the example there is legal. If you want to options explicit then sure, I'll add them.

Reply:
Good point. I also updated the example slightly to use the `finalize` callback.